maven/.gitignore

```diff
@@ -1 +0,0 @@
-/target/
```

Comment:
Why are you removing this? The `maven/target` folder must not be under version control.